lib/view_component/test_helpers.rb

```diff
@@ -38,7 +38,6 @@ def refute_component_rendered
     # assert_text("Hello, World!")
     # ```
     #
-    # @param component [ViewComponent::Base, ViewComponent::Collection] The instance of the component to be rendered.
```

Comment:
I think we want to leave this line in ❤️

Reply:
Woops, thanks!